docker/arm/Dockerfile.arm32v7

```diff
@@ -1,7 +1,6 @@
 # Dockerfile to be used in an ARMv7 environment like Raspverry PI 4
-
-FROM arm32v7/node:dubnium-buster-slim
-MAINTAINER Timo Knapp <mail@timoknapp.com>
+FROM arm32v7/node:slim
+LABEL maintainer="Coderaiser, tea-mo903"
 
```

Comment:
Why don't you reuse existing `Dockerfile` :)?

Reply:
I tried at first, but the problem with that is, that you have to run the `docker build` command from the corresponding `os architecture` then. I can only do it for `arm32` and  `amd64` then. So its more robust and reliable to explicitly define the image for the desired architecture. Then the `docker build` works just fine

Reply:
@coderaiser what do you think about it? Do you have any other open topic regarding the PR?